docs/source/_static/style.css

```diff
@@ -0,0 +1,3 @@
+.wy-nav-content {
+    max-width: 1200px !important;
```

Comment:
@tomvothecoder The width of https://e3sm-project.github.io/e3sm_data_docs/_build/html/simulation_locations.html is pretty narrow. Much of the table is obscured.

I seem to remember needing to do something on `zppy` to widen the docs. I tried adding https://github.com/E3SM-Project/zppy/blob/main/docs/source/_static/style.css#L2, but this doesn't seem to work.

Do you have any ideas?

Reply:
The `style.css` file needs to be added to `conf.py` for it to register the CSS updates.

I just pushed a commit for this.